### PR TITLE
[DOC] Documentation updates for release 24.02

### PR DESCRIPTION
**Pull request by cjnolet**

### CODE REVIEW

File: docs/source/vector_search_tutorial.md [hunk 1]
```diff
@@ -216,6 +216,55 @@ While it may seem like unnecessary additional work to wrap the result in a `host
 
 The following sections present some advanced features that we have found can be useful for squeezing more utilization out of GPU hardware. As you've seen in this tutorial, RAFT provides several very useful tools and building blocks for developing accelerated applications beyond vector search capabilities.
 
+### Serialization
```
Comment (dantegd) [inline docs/source/vector_search_tutorial.md:219]: The tutorial has become kinda large, I wonder if an index at the beginning of the sections would be helpful, what do you think?

**Review by dantegd — APPROVED** [review]
Changes look great, had a small suggestion

File: docs/source/vector_search_tutorial.md (hunk 1 above)
Comment (cjnolet) [reply docs/source/vector_search_tutorial.md:219]: Like a table of contents? sure we can do that

**Pull request merged**